servers/kaorynetwork/metadata.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "kaorynetwork",
+    "name": "Kaory Network",
+    "description": "Minecraft network with multiple minigames, multi-language and multi-region.",
+    "addresses": [
```

Comment:
You don't need to include the subdomain in the addresses array. You only need the domain.tld